sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations/GalleryApplicationVersionPublishingProfile.cs

```diff
@@ -19,5 +19,16 @@ public GalleryApplicationVersionPublishingProfile(UserArtifactSource source, ILi
             EnableHealthCheck = enableHealthCheck;
             CustomInit();
         }
+
+        public GalleryApplicationVersionPublishingProfile(UserArtifactSource source, IList<TargetRegion> targetRegions, int? replicaCount, bool? excludeFromLatest, System.DateTime? publishedDate, System.DateTime? endOfLifeDate, string storageAccountType, string replicationMode, UserArtifactManage manageActions, bool? enableHealthCheck = default(bool?), UserArtifactSettings settings = default(UserArtifactSettings), IDictionary<string, string> advancedSettings = default(IDictionary<string, string>))
```

Comment:
Based on the wiki [example](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/182581/Creating-Customization-Constructor?anchor=how-to-create-custom-constructors), I don't think you need to append the new parameters in the custom construtor if I understand correctly.

Reply:
The custom constructor was added to address the comment left by @Sandido. New parameters should aways come after existing parameters. Auto generation of SDK re-ordered them. So I added a custom constructor to address that.

Reply:
My understanding is that you don't need to add the new parameters (settings, advancedSettings) in the custom construtor as it's used to avoid breaking changes for code written with the previous version of generated construtor. If customers need to set settings or advancedSettings, they should use the new generated constructor. @Sandido can you help confirm it?

Reply:
@D1v38om83r , Feng is correct here. As per the [wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/182581/Creating-Customization-Constructor?anchor=how-to-create-custom-constructors),  the new parameters Settings and AdvancedSettings should not be in the new customization constructor. 
Please correct this customization.

Reply:
Customization corrected. Also accounted for targetExtendedLocations that was added in a separate PR.